README.md

```diff
@@ -1,5 +1,161 @@
 # Datadog SDK for Go testing
-Datadog Test Instrumentation framework for Go
+> Datadog Test Instrumentation framework for Go
 
-# Status
-Work in progress
\ No newline at end of file
+## Getting Started
+
+### Installing
+Installation of the DataDog go testing sdk is done via `go get`:
+
+```shell
+go get -u github.com/DataDog/dd-sdk-go-testing
+```
+
+#### Requires:
+- Go >= 1.12
+- Datadog's Trace Agent >= 5.21.1
+
+### Instrumenting your tests
+In order to instrument your tests that use Go's native 
+[`testing`](https://golang.org/pkg/testing/) package, you have to 
+call `ddtesting.Run(m)` in your `TestMain` function, and call
+`ddtesting.StartTest(t)` or `ddtesting.StartTestWithContext(ctx, t)`
+and `defer finish()` on each test.
+
+For example:
+
+```go
+package go_sdk_sample
+
+import (
+	"os"
+	"testing"
+
```

Comment:
is this empty line intended?

Reply:
yes, to make the separation between internal packages and external modules.